requirements.txt

```diff
@@ -1,3 +1,4 @@
 alsaaudio
 pafy
 mutagen
+soundcloud-lib
```

Comment:
Is the compiled packages, i.e., do we need to push them to rubygem, or is it a python only lib?

Reply:
@norbusan , it's just a pypi package. We can install it directly via pip

Reply:
THanks